js/data/bucket.js

```diff
@@ -223,7 +225,7 @@ Bucket.prototype.setAttribPointers = function(programName, gl, program, offset,
     var disabledAttributes = this.attributes[programName].disabled;
     for (var i = 0; i < disabledAttributes.length; i++) {
         attribute = disabledAttributes[i];
-        if (attribute.isLayerConstant === false && layer.id !== attribute.layerId) continue;
+        assert(attribute.paintProperty);
```

Comment:
fixed in f469330d8aeb9d5db5bb7c11cac7b347b6cdbed2

it's safe to remove the `attribute.isLayerConstant === false` because this is always true for disabled attributes, right?